src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkTest.java

```diff
@@ -213,4 +213,10 @@ public void testResolverCascading() {
     CloudSdk sdk = builder.build();
     assertEquals("r1 should not resolve", r2.getCloudSdkPath(), sdk.getSdkPath());
   }
+
+  @Test
+  public void testGetJavaBinary() {
```

Comment:
Fixed. On the topic of failing tests in Windows, I noticed there's a bunch of others failing.
@meltsufin maybe we could look into turning on AppVeyor for appengine-plugins-core too?